sdcm/cluster.py

```diff
@@ -2646,11 +2646,11 @@ def get_nodes_status(self):
                         if node_ip:
                             LOGGER.error("Get nodes statuses. Failed to find a node in cluster by IP: %s", node_ip)
 
-        except Exception as exc:  # pylint: disable=broad-except
+        except Exception as error:  # pylint: disable=broad-except
             ClusterHealthValidatorEvent.NodeStatus(
                 severity=Severity.WARNING,
                 node=self.name,
-                message=f"Unable to get nodetool status from `{self.name}': {exc}",
+                message=f"Unable to get nodetool status from `{self.name}': {error=}",
```

Comment:
how this is related to this change ?

Reply:
This is something I hit into while looking at the logs. This is not related to the main purpose of the change, so I put it into a separate commit...

Reply:
I updated the PR, so I believe this discussion is not relevant anymore